### PR TITLE
feat: add timer orientation

### PR DESCRIPTION
**Pull request by solaolateju**
**Horizontal view of timer**

Screenshot 2020-04-06 at 17 01 23

**Vertical view of timer**

Screenshot 2020-04-09 at 10 03 13

### CODE REVIEW

File: src/components/modes/student/StudentView.js [hunk 1]
```diff
@@ -28,75 +27,72 @@ const useStyles = makeStyles((theme) => ({
...
+            <Grid container spacing={3} style={{ width: '100%' }}>
               <Grid item xs={12} align="center">
                 <FormControl component="fieldset">
                   <FormLabel component="legend">Choose Timer Type</FormLabel>
-                  <RadioGroup aria-label="selectTimerType" name="gender1" value={value} onChange={handleChange}>
-                    <FormControlLabel value="forward" control={<Radio />} label="Count Up" onClick={() => setValue("forward")} />
-                    <FormControlLabel value="backward" control={<Radio />} label="Count Down " onClick={() => setValue("backward")} />
+                  <RadioGroup
+                    aria-label="selectTimerType"
+                    name="gender1"
```
Comment (juancarlosfarah) [inline src/components/modes/student/StudentView.js:65]: gender? Looks like you forgot to update the `name` property.

File: src/components/modes/student/StudentView.js [hunk 2]
```diff
@@ -28,75 +27,72 @@ const useStyles = makeStyles((theme) => ({
...
+        initialTime={value === 'forward' ? 0 : 4800000}
         startImmediately
-
-
         direction={value}
       >
         {({ start, pause, reset }) => (
           <div className={classes.root}>
-            <Grid container spacing={3} style={{width: "100%"}}>
+            <Grid container spacing={3} style={{ width: '100%' }}>
```
Comment (juancarlosfarah) [inline src/components/modes/student/StudentView.js:59]: The style tag should be avoided. Instead, pass a `className={classes.gridContainer}` and add the styles in the styles object.

File: src/components/modes/student/StudentView.js [hunk 3]
```diff
@@ -28,75 +27,72 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-// const timerType = "backward"
-// const startValue = timerType === "forward" ? 0 : 4800000
+const GridItem = ({ timeValue, timeUnit }) => (
```
Comment (juancarlosfarah) [inline src/components/modes/student/StudentView.js:30]: Why don't you name this `Counter` and put it in a separate file?

File: src/components/modes/student/StudentView.js [hunk 4]
```diff
@@ -28,75 +27,72 @@ const useStyles = makeStyles((theme) => ({
...
+  <Grid item xs={3} style={{ width: '100%' }}>
```
Comment (juancarlosfarah) [inline src/components/modes/student/StudentView.js:31]: The style tag should be avoided. Instead, pass a `className={classes.gridItem}` and add the styles in the styles object.

File: src/components/modes/student/StudentMode.js [hunk 5]
```diff
@@ -51,7 +51,7 @@ class StudentMode extends Component {
       case FEEDBACK_VIEW:
       case DEFAULT_VIEW:
       default:
-        return <StudentView />;
+        return <StudentView orientation="horizontal" />;
```
Comment (juancarlosfarah) [inline src/components/modes/student/StudentMode.js:54]: We want this to be dynamic. I can walk you through this later.

**Comment by juancarlosfarah** [comment]
@solaolateju, for layout and visual changes, it's nice to include a screenshot in the PR, within a comment.

**Comment by solaolateju** [comment]
Horizontal view

![Vertical view(https://user-images.githubusercontent.com/18397749/78877491-3a766a00-7a51-11ea-8647-00dbc9f7b04a.png)

**Comment by solaolateju** [comment]
![Vertical view(https://user-images.githubusercontent.com/18397749/78877491-3a766a00-7a51-11ea-8647-00dbc9f7b04a.png)

**Comment by solaolateju** [comment]
Screenshot 2020-04-09 at 10 03 13

**Pull request merged**